res/values-in/strings.xml

```diff
@@ -1036,7 +1036,7 @@
     <string name="Enable_Profile_Colors">Aktifkan warna profil </string>
     <string name="Session_Stats">Session Stats</string>
     <string name="Time_">Waktu:</string>
-    <string name="Show_Session_Stats">Show Session Stats</string>
+    <string name="Show_Session_Stats">Tampilkan Sessi Stats</string>
```

Comment:
At where I know about the Indonesian grammatical rules, I don't think "Sessi" uses double "s" for the words, except for a greeting word

Reply:
yes i was wrong can i fix it again?

Reply:
U can ^^

Reply:
ok i have fixed it, is it correct?